farcy/handlers.py

```diff
@@ -245,3 +245,34 @@ def _process(self, filename):
             retval[offense['location']['line']].append(
                 '{cop_name}: {message}'.format(**offense))
         return retval
+
+
+class SCSSLint(ExtHandler):
+
+    """Provides feedback for css and scss files using scss-lint"""
+
+    BINARY = 'scss-lint'
+    BINARY_VERSION = '0.43.2'
+    EXTENSIONS = ['.css', '.scss']
+
+    def _process(self, filename):
+        command = [self.BINARY, '-f', 'JSON']
+        config_path = self.config_file_path
+        if config_path:
+            command += ['-c', config_path]
+
+        data = json.loads(self.execute(command + [filename]))
+
+        retval = defaultdict(list)
+        if data.values() == []: return retval
```

Comment:
I think the convention for empty list check is just

```
if not data.values()
  return retval
```

But it's been a long time since I've written Python!